website_field_autocomplete/static/src/js/field_autocomplete.js

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2016 LasLabs Inc.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+odoo.define('website_field_autocomplete.field_autocomplete', function(require){
+  "use strict";
+
+  var snippet_animation = require('web_editor.snippets.animation');
+  var Model = require('web.Model');
+
+  snippet_animation.registry.field_autocomplete = snippet_animation.Class.extend({
+
+    selector: '.s_website_autocomplete',
```

Comment:
Will change - Out of curiosity, is this an Odoo module standard in order to not conflict with Odoo core namespacing? I used the `s_` because it seems that's what Odoo SA is using such as `s_website_form`